app/pages/lab/workflows-container.jsx

```diff
@@ -50,6 +53,33 @@ export default class WorkflowsContainer extends React.Component {
     }
   }
 
+  handleSetStatsCompletenessType(e, page, workflow) {
+    workflow.update({ 'configuration.stats_completeness_type': e.target.value }).save()
+      .catch(error => console.log(error))
+      .then(() => this.getWorkflowList(page));
+  }
+
+  handleWorkflowStatusChange(e, page, workflow) {
+    const defaultWorkflow = (this.props.configuration) ? this.props.configuration.default_workflow : null;
```

Comment:
I think this should be `const defaultWorkflow = (this.props.configuration && this.props.configuration.default_workflow) ? this.props.configuration.default_workflow : null;`

Also, could the prop be renamed projectConfiguration for clarity? I had to read this line a couple of times because I thought initially it was referring to the workflow configuration object.